releasenotes/notes/remove-pulse-passes-3128f27ed7e42bf6.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade_transpiler:
+  - |
+    The ``PulseGates``, ``ValidatePulseGates``, ``RXCalibrationBuilder``, ``RZXCalibrationBuilder``,
+    ``RZXCalibrationBuilderNoEcho`` and ``EchoRZXWeylDecomposition`` passes have been removed,
+    following their deprecation in Qiskit 1.3. These passes depend on and relate to the Pulse
+    package which is also being removed in Qiskit 2.0.
```

Comment:
Lint is indicating that the `inst_map` argument in `generate_scheduling` is no longer used, if we were to remove it, the reno should also mention it.

Reply:
Yes, the argument is no longer used within `generate_scheduling` itself indeed (and I addressed the lint warning about that in this PR), but I was thinking about mentioning it in a different reno bullet that removes `inst_map` from all the other functions (e.g. `generate_preset_pass_manager`), together possibly with more removals in the transpiler/circuit area rather than here. So if you're OK, I would keep this reno bullet about passes